.eslintignore

```diff
@@ -0,0 +1,5 @@
+imports/api/*.js
+imports/ui/js/
+imports/ui/*.jsx
+imports/ui/*.js
+server/
```

Comment:
These are not suppose to be ignored! They are ignored right now to make the linter CI job passing! They should eventually be removed from this file.